packages/flutter_test/lib/src/binding.dart

```diff
@@ -857,16 +938,37 @@ class _LiveTestPointerRecord {
 
 class _LiveTestRenderView extends RenderView {
   _LiveTestRenderView({
-    ViewConfiguration configuration
+    ViewConfiguration configuration,
+    this.onNeedPaint,
   }) : super(configuration: configuration);
 
   @override
   TestViewConfiguration get configuration => super.configuration;
   @override
   set configuration(covariant TestViewConfiguration value) { super.configuration = value; }
 
+  final VoidCallback onNeedPaint;
+
   final Map<int, _LiveTestPointerRecord> _pointers = <int, _LiveTestPointerRecord>{};
 
+  TextPainter _label;
+  static const TextStyle _labelStyle = const TextStyle(
+    fontFamily: 'sans-serif',
+    fontSize: 10.0,
+  );
+  void _setDescription(String value) {
+    assert(value != null);
+    if (value.isEmpty) {
+      _label = null;
+      return;
+    }
+    _label ??= new TextPainter(textAlign: TextAlign.left);
+    _label.text = new TextSpan(text: value, style: _labelStyle);
```

Comment:
Does that work right with `??=`? I want to set the text property even when `_label` is already constructed.